tests/comfy_cli/command/test_code_search.py

```
@@ -323,6 +334,60 @@ def test_limit_hit_message(self, capsys, limit_hit_search):
+        output = capsys.readouterr().out
+        # File URL printed once per file (2 files in fixture).
+        assert output.count("https://github.com/Comfy-Org/ComfyUI/blob/abc123def456/nodes.py") == 1
+        assert "blob/abc123def456/nodes.py" in output
+        assert "blob/abc123def456/server.py" in output
+        # Per-line anchors must NOT appear in non-TTY mode.
+        assert "#L42" not in output
+        assert "#L56" not in output
+        # No OSC 8 escape sequences.
+        assert "\x1b]8;" not in output
+
+    def test_tty_emits_osc8_and_hides_urls(self, search_response):
```

Pylint (4.0.5)

[warning] Redefining name 'search_response' from outer scope (line 40)

(W0621)